### DISCUSSION TITLE
shell frosting löschen

### DISCUSSION THREAD

**Discussion by mwcast** [community/community#83059] [Accessibility]
### Select Topic Area

Question

### Body

Wie kann ich den Ordner [Shell Frosting 2023.508.23451.1 für Windows] wieder von meinem PC löschen? Ich erhalte immer die Meldung "Zugriff verweigert" wenn ich die "frosting.dll" löschen will.

**Comment by Akash1134** [comment]
Hi @mwcast , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
